bootstrap/pkg/kfapp/coordinator/coordinator.go

```diff
@@ -76,7 +76,10 @@ func downloadToCache(platform string, appDir string, version string, useBasicAut
 	cacheDir := path.Join(appDir, kftypes.DefaultCacheDir)
 	// idempotency
 	if _, err := os.Stat(cacheDir); !os.IsNotExist(err) {
-		os.RemoveAll(cacheDir)
+		rmErr := os.RemoveAll(cacheDir)
```

Comment:
is this gonna be used by click to deploy app?  if it is, need to be careful of thread safety.

@kunmingg

Reply:
What's the threading issue here? 

My expectation is each request in the deploy app uses a separate instance of KfApp and uses a different directory to copy down the configs to.

So even if we simultanteously handle two duplicate requests for the same app; both would be modifying a different directory.

Reply:
yea, using different directory would be fine.  just wanna make sure it's working that way.

Reply:
Shouldn't affect logic of deploy app. thanks